dotnet/src/webdriver/Internal/Logging/FileLogHandler.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+using System.IO;
+
+namespace OpenQA.Selenium.Internal.Logging
+{
+    /// <summary>
+    /// Represents a log handler that writes log events to a file.
+    /// </summary>
+    public class FileLogHandler : ILogHandler, IDisposable
+    {
+        // performance trick to avoid expensive Enum.ToString() with fixed length
+        private static readonly string[] _levels = { "TRACE", "DEBUG", " INFO", " WARN", "ERROR" };
+
+        private FileStream _fileStream;
+        private StreamWriter _streamWriter;
+
+        private readonly object _lockObj = new object();
+        private bool _isDisposed;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="FileLogHandler"/> class with the specified file path.
+        /// </summary>
+        /// <param name="path">The path of the log file.</param>
+        public FileLogHandler(string path)
```

Comment:
Maybe rename `path` parameter to `filePath`? Just to be certain that it is not a directory path.

Reply:
Done.